dcos/dcos-ui
# DCOS-56347: refactor add service button

## Testing
1. Go to the Services page
2. Click the "+" in the page header

A dropdown should appear with "Run a Service" and "Create Group", and there should be no overflow button on the right where "Create Group" used to be

## Trade-offs
We're waiting to implement the exact design ([InVision mockup](https://mesosphere.invisionapp.com/d/main#/console/17664445/366217481/preview)) until the design team can have some time to consider a dropdown add button style for all of DC/OS.

## Dependencies



## Screenshots
<img width="1147" alt="Screen Shot 2019-07-17 at 8 24 57 PM" src="https://user-images.githubusercontent.com/2313998/61421377-e29d3f00-a8d4-11e9-8ee2-67a44b038744.png">
